Pager/Doctrine/Pager.php

```diff
@@ -21,10 +21,13 @@
  */
 class Pager extends BasePager
 {
+    /**
+     * @deprecated This property is deprecated since version 2.2 and will be removed in 3.0.
```

Comment:
Well, no stable version yet for this bundle.

This may be removed directly.

ping @rande